app/helpers/sessions_helper.rb

```diff
@@ -0,0 +1,20 @@
+module SessionsHelper
+  def log_in user
+    session[:user_id] = user.id
+  end
+
+  def current_user
+    if session[:user_id]
+      @current_user ||= User.find_by id: session[:user_id]
```

Comment:
`||=` là gì em

Reply:
Dạ toán tử ||= cón nghĩa là "or equals", dùng để gán giá trị cho biến kèm theo điều kiện, nếu x là nil/false/underfined --> gán cho vế bên trái, ngược lại x = chính nó.
x | |= 1   <=>  x = x | | 1